x/gov/abci_test.go

```diff
@@ -4,7 +4,7 @@ import (
 	"testing"
 	"time"
 
-	"cosmossdk.io/math"
+	sdkmath "cosmossdk.io/math"
```

Comment:
is there a need for this change?

Reply:
Na, i just thought maybe whilst im changing all, make them consistent